app/sdil/config/SDILComponents.scala

```diff
@@ -70,4 +68,7 @@ class SDILComponents(context: Context)
   override val appName = configuration.get[String]("appName")
 
   override lazy val metrics: Metrics = wire[MetricsImpl]
+
+  override def httpFilters: Seq[EssentialFilter] = Nil
```

Comment:
I wonder if this will create issues now that you specify it as Nil? Is it possible to not override it at all?

Reply:
As we created `SDILComponents` object, we cannot keep it as abstract class. Is it okay to leave `SdilFilters` as is without removing. I deleted this because it uses `FrontendFilters` which is deprecated now.